cmd/groundstation/listPlans.go

```diff
@@ -16,6 +16,7 @@ package groundstation
 
 import (
 	"fmt"
+	"github.com/infostellarinc/stellarcli/cmd/util"
```

Comment:
We are currently using an import style with `built in`, `third-party`, `infostellar` groups. I think it'd also be ok to have `built in` and `github` groups since it is a bit annoying, but let's pick and be consistent throughout.